src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.spec.ts

```diff
@@ -126,6 +132,50 @@ describe('PaypalCommerceButtonStrategy', () => {
         expect(paypal.Buttons).toHaveBeenCalledWith({
             createOrder: expect.any(Function),
             onApprove: expect.any(Function),
+            style: paypalOptions.style,
+        });
+    });
+
+    it('validate style for PaypalCommerce checkout button', async () => {
+        const style = {
```

Comment:
can you do `const style as PaypalButtonStyleOptions` to avoid the casting on line 150?

Reply:
I can't use in this case `PaypalButtonStyleOptions` because I would like to pass invalid data for check function of validation. Maybe in this case will be better `style: any` ?